frontend/src/pages/team/Library/Blueprints.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <ul v-if="blueprints.length" class="flow-categories-wrapper">
+        <li v-for="(flowBlueprints, category) in blueprintsByCategory" :key="category" class="category">
+            <h2 class="title">{{ category }}</h2>
+            <div class="tiles-wrapper">
+                <BlueprintTile
+                    v-for="(blueprint, index) in flowBlueprints"
+                    :key="index"
+                    class="blueprint-tile"
+                    :blueprint="blueprint"
+                    :data-el="blueprint.id"
+                    @selected="onBlueprintSelect"
+                />
+            </div>
+        </li>
+    </ul>
+    <EmptyState v-else :featureUnavailable="!isSharedLibraryFeatureEnabledForPlatform" :featureUnavailableToTeam="!isSharedLibraryFeatureEnabledForTeam">
+        <template #img>
+            <img src="../../../images/empty-states/team-library.png" alt="team-logo">
+        </template>
+        <template #header>Create your own Blueprints</template>
+        <template #message>
+            <p>
+                Your Blueprints will be shown here, and will be available within all of your Node-RED instances on FlowFuse.
+            </p>
+        </template>
+        <template #actions>
+            <ff-button v-if="isSharedLibraryFeatureEnabled" :to="{name: 'AdminFlowBlueprints'}" data-el="go-to-blueprints">
+                Go To Blueprints
+            </ff-button>
+            <ff-button v-else :to="{name: 'AdminFlowBlueprints'}" :disabled="true">
+                Add To Library
```

Comment:
these actions should only show for Platform Admin users?

Reply:
Tbh I just mirrored the TeamLibrary functionality without giving it a second thought (frontend/src/pages/team/Library/TeamLibrary.vue:38). Given that you need the same set of permissions for both blueprints and team library I didn't alter anything but the link's location

Reply:
As a non-admin, the "Go to Blueprints" button needs to be hidden, as I won't have access to it. In the Team Library page, that link goes to the Instances page, which non-admins _can_ access.

This should then get checked by an E2E test, utilising our different user levels available in Cypress.

Reply:
The template header might need addressing, end users might get mislead by 'Create your own Blueprints'

Reply:
Agreed - let's change to "No Blueprints Available" if you're not a Platform admin. Then the messaging to be around "Speak to your platform admin to add some blueprints" for non-platform